patch/llpcPatchBufferOp.cpp

```diff
@@ -1531,15 +1531,15 @@ Value* PatchBufferOp::ReplaceLoad(
 
     const DataLayout& dataLayout = m_pBuilder->GetInsertBlock()->getModule()->getDataLayout();
 
-    Type* const pLoadType = pLoadInst->getType();
+    Type* const loadType = pLoadInst->getType();
```

Comment:
loadType -> pLoadType
Current style still add "p" prefix for pointer variables.